app/views/shared/_feeds.html.erb

```diff
@@ -1,14 +1,8 @@
 <% unless atom_url.blank? %>
-  <div class="feeds">
-    <span class="govuk-body" <%= t_lang("feeds.get_updates_to_this_list") %>><%= t("feeds.get_updates_to_this_list") %></span>
-    <%= render "govuk_publishing_components/components/subscription_links", {
-      hide_heading: true,
-      email_signup_link: email_signup_path(atom_url),
-      email_signup_link_text: t('feeds.email'),
-      email_signup_link_text_locale: t_fallback('feeds.email'),
-      feed_link_box_value: atom_url,
-      feed_link_text: t("feeds.feed"),
-      feed_link_text_locale: t_fallback("feeds.feed"),
-    } %>
-  </div>
+  <%= render "govuk_publishing_components/components/subscription_links", {
+    email_signup_link: email_signup_path(atom_url),
```

Comment:
The Content / capitalisation on [this page](https://www.integration.publishing.service.gov.uk/government/topical-events/2022-events-platinum-jubilee-commonwealth-games-festival-uk) ideally should read

"Get emails" 
"Subscribe to feed"

---
![image](https://user-images.githubusercontent.com/71266765/135231682-7b473aeb-8083-4bc9-ab2a-c5234629e45a.png)

Reply:
Thanks Chris, _I think_ the subscription links component should already be configured with these labels.

I checked against this page https://www.integration.publishing.service.gov.uk/government/topical-events/2022-events-platinum-jubilee-commonwealth-games-festival-uk. Also included screenshot here:

<img src="https://user-images.githubusercontent.com/87758239/135257103-950f6883-64ba-44b0-979f-4732b9906b05.png" width="400">

Although not sure if there is another usage on the page I'm missing?

Reply:
Ah sweet can see this now